### ISSUE TITLE
Lobart thinks you stole something from his chest the first time you talk to him

### ISSUE THREAD

**Issue by degenerated1123** [REGoth-project/REGoth#118]

**Comment by degenerated1123** [comment]
External `Mob_HasItems` needs to be implemented. Regarding script-code:

```cpp
func int DIA_Lobart_STOLENCLOTHS_Condition ()
{
	if (Mob_HasItems("CHEST_LOBART", ITAR_Bau_L) == FALSE)
	&& (Lobart_Kleidung_Verkauft == FALSE)
	&& (hero.guild == GIL_NONE)
	{
		return TRUE;
	};
};
```

**Comment by degenerated1123** [comment]
Fixed in PR #122

**Comment by markusobi** [comment]
Well now he never rants about the stolen item, because looting the chest copies the items. I can't find it in the code. Could you implement moving items when looting (chests, npcs) rather than copying?

**Comment by degenerated1123** [comment]
That would be in `src/logic/mobs/Container.cpp`, line 53. Adding a simple `m_Contents.clear();` after giving the items should be enough.

Am working on the animation-system by @astillich at the moment ;)

**Comment by ChaosMarc** [comment]
The suggested solution has been implemented in 9074734b566ea74030eafff30ead1e45d98f46a2
I guess this issue can be closed now
